leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/LeakTraceWordWrapper.kt

```diff
@@ -0,0 +1,368 @@
+package leakcanary.internal.activity.screen
+
+import java.util.Stack
+
+/**
+ * Performs word wrapping of leak traces.
+ */
+object LeakTraceWordWrapper {
+
+  private const val SPACE_CHARACTER = '\u0020'
+  private const val TILDE_CHARACTER = '\u007E'
+  private const val PERIOD_CHARACTER = '\u002E'
+  private const val BREAKING_CHARACTER_LENGTH = 1
+  const val BREAK_CHARACTER = '\n'
+
+  /**
+   * This implements a greedy wrapping algorithm.
+   *
+   * <p>
```

Comment:
Nit: keep an empty line between the two but remove `<p>` to be consistent with javadoc style elsewhere in the repo.